playbooks/roles/mongodb/tasks/main.yml

```diff
@@ -30,6 +30,15 @@
   with_items:
     - 27017
 
+- name: configure log rotation for the MongoDB logs
+  copy:
+    template:
+      src: logrotate.conf.j2
+      dest: /etc/logrotate.d/mongodb.conf
+      owner: root
+      group: root
```

Comment:
`root` is the default `owner` and `group`, so these can be erased to keep it succinct.